src/sif/bicyclecost.cc

```diff
@@ -399,9 +398,7 @@ class BicycleCost : public DynamicCost {
    * mode used by the costing method. It's also used to filter
    * edges not usable / inaccessible by bicycle.
    */
-  float Filter(const baldr::DirectedEdge* edge,
-               const baldr::GraphId& /*edgeid*/,
-               const baldr::GraphTile* /*tile*/) const override {
+  float Filter(const baldr::DirectedEdge* edge, const baldr::GraphTile*) const override {
```

Comment:
that's great! I didn't notice that we `edgeid` is redundant here . it makes the code much easier)

Reply:
at first, i didnt think of it either or i would have suggested it. sorry for not coming up with it sooner actually!